2020-04-24 01:54:56,058 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 01:54:56,059 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 01:54:56,059 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 01:55:03,250 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 01:55:03,250 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 01:55:03,250 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 01:55:19,342 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 01:55:19,342 - case - DEBUG - 测试数据 - [logger.py:33] 
2020-04-24 01:55:19,343 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 01:55:19,343 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 02:15:37,306 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 02:15:37,306 - case - DEBUG - 测试数据 - [logger.py:33] 
2020-04-24 02:15:37,307 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 02:15:37,307 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 02:17:16,514 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 02:17:16,515 - case - DEBUG - 测试数据 - [logger.py:33] 
2020-04-24 02:17:16,515 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 02:17:16,515 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 02:19:24,834 - case - INFO - 测试开始啦 - [logger.py:32] 
2020-04-24 02:19:24,834 - case - DEBUG - 测试数据 - [logger.py:33] 
2020-04-24 02:19:24,834 - case - ERROR - 测试报错 - [logger.py:34] 
2020-04-24 02:19:24,834 - case - INFO - 测试结束啦 - [logger.py:35] 
2020-04-24 02:19:26,109 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 02:19:26,110 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 02:19:26,110 - API_4.common.http_request - DEBUG - 请求的url:http://test.lemonban.com/futureloan/mvc/api/member/login - [http_request.py:44] 
2020-04-24 02:19:26,111 - API_4.common.http_request - DEBUG - 请求data:{'mobilephone': '15810447878', 'pwd': '123456'} - [http_request.py:45] 
2020-04-24 02:19:26,427 - API_4.common.http_request - DEBUG - 请求的response:{"status":1,"code":"10001","data":null,"msg":"登录成功"} - [http_request.py:57] 
2020-04-24 02:19:26,489 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 02:19:26,489 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 02:19:26,489 - API_4.common.http_request - DEBUG - 请求的url:http://test.lemonban.com/futureloan/mvc/api/member/login - [http_request.py:44] 
2020-04-24 02:19:26,490 - API_4.common.http_request - DEBUG - 请求data:{'mobilephone': '15810447878', 'pwd': '098786'} - [http_request.py:45] 
2020-04-24 02:19:26,696 - API_4.common.http_request - DEBUG - 请求的response:{"status":0,"code":"20111","data":null,"msg":"用户名或密码错误"} - [http_request.py:57] 
2020-04-24 02:19:26,741 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 02:19:26,741 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 02:19:26,742 - API_4.common.http_request - DEBUG - 请求的url:http://test.lemonban.com/futureloan/mvc/api/member/login - [http_request.py:44] 
2020-04-24 02:19:26,742 - API_4.common.http_request - DEBUG - 请求data:{'mobilephone': '15810447878', 'pwd': ''} - [http_request.py:45] 
2020-04-24 02:19:26,788 - API_4.common.http_request - DEBUG - 请求的response:{"status":0,"code":"20103","data":null,"msg":"密码不能为空"} - [http_request.py:57] 
2020-04-24 02:19:26,830 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 02:19:26,830 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 02:19:26,830 - API_4.common.http_request - DEBUG - 请求的url:http://test.lemonban.com/futureloan/mvc/api/member/login - [http_request.py:44] 
2020-04-24 02:19:26,830 - API_4.common.http_request - DEBUG - 请求data:{'mobilephone': '', 'pwd': '123456'} - [http_request.py:45] 
2020-04-24 02:19:26,870 - API_4.common.http_request - DEBUG - 请求的response:{"status":0,"code":"20103","data":null,"msg":"手机号不能为空"} - [http_request.py:57] 
2020-04-24 02:19:26,906 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 02:19:26,907 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 02:34:24,824 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 02:34:24,825 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 02:34:25,133 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 02:34:25,133 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 02:34:25,371 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 02:34:25,372 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 02:34:25,451 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 02:34:25,451 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 02:34:25,527 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 02:34:25,528 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 02:46:42,064 - API_4.testcases.test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 02:46:42,065 - API_4.testcases.test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 02:46:42,370 - API_4.testcases.test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 02:46:42,370 - API_4.testcases.test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 02:46:42,607 - API_4.testcases.test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 02:46:42,607 - API_4.testcases.test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 02:46:42,694 - API_4.testcases.test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 02:46:42,694 - API_4.testcases.test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 02:46:42,775 - API_4.testcases.test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 02:46:42,775 - API_4.testcases.test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 02:49:07,600 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 02:49:07,600 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 02:49:07,888 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 02:49:07,888 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 02:49:08,109 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 02:49:08,110 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 02:49:08,191 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 02:49:08,192 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 02:49:08,271 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 02:49:08,272 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 02:55:44,440 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 02:55:44,441 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 02:55:44,728 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 02:55:44,729 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 02:55:44,971 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 02:55:44,972 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 02:55:45,054 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 02:55:45,055 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 02:55:45,136 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 02:55:45,137 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 03:00:58,934 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 03:00:58,934 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 03:00:59,264 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 03:00:59,265 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 03:00:59,520 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 03:00:59,520 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 03:00:59,614 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 03:00:59,614 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 03:00:59,688 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 03:00:59,688 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
2020-04-24 13:45:17,498 - test_login - INFO - 准备测试前置 - [test_login.py:24] 
2020-04-24 13:45:17,499 - test_login - INFO - 开始测试:正常登录 - [test_login.py:29] 
2020-04-24 13:45:17,858 - test_login - INFO - 结束测试:正常登录 - [test_login.py:38] 
2020-04-24 13:45:17,858 - test_login - INFO - 开始测试:输入错误的密码 - [test_login.py:29] 
2020-04-24 13:45:18,106 - test_login - INFO - 结束测试:输入错误的密码 - [test_login.py:38] 
2020-04-24 13:45:18,107 - test_login - INFO - 开始测试:不输入密码 - [test_login.py:29] 
2020-04-24 13:45:18,214 - test_login - INFO - 结束测试:不输入密码 - [test_login.py:38] 
2020-04-24 13:45:18,215 - test_login - INFO - 开始测试:不输入手机号码 - [test_login.py:29] 
2020-04-24 13:45:18,379 - test_login - INFO - 结束测试:不输入手机号码 - [test_login.py:38] 
2020-04-24 13:45:18,379 - test_login - INFO - 测试后置处理 - [test_login.py:42] 
